### PR TITLE
Move '/etc/sysconfig/*' parsers to sysconfig.py

### PR DESCRIPTION
**Pull request by xiangce**
- Only move the parsers inherited from SysconfigOptions
- Mark the old parsers as deprecated

### CODE REVIEW

File: insights/parsers/sysconfig.py [hunk 1]
```diff
@@ -24,36 +36,76 @@
 LibvirtGuestsSysconfig - file ``/etc/sysconfig/libvirt-guests``
 ---------------------------------------------------------------
 
+MemcachedSysconfig - file ``/etc/sysconfig/memcached``
+------------------------------------------------------
+
 MongodSysconfig - file ``/etc/sysconfig/mongod``
 ------------------------------------------------
 
+NetconsoleSysconifg -file ``/etc/sysconfig/netconsole``
```
Comment (bfahr) [inline insights/parsers/sysconfig.py]: Should be `NetconsoleSysconfig`.

File: insights/parsers/sysconfig.py [hunk 2]
```diff
@@ -275,61 +360,50 @@ class MongodSysconfig(SysconfigOptions):
     pass
 
 
-@parser(Specs.sysconfig_memcached)
-class MemcachedSysconfig(SysconfigOptions):
-    """
-    A parser for ``memcached`` configuration, stored in
-    ``/etc/sysconfig/memcached``.
+@parser(Specs.netconsole)
+class NetconsoleSysconifg(SysconfigOptions):
```
Comment (bfahr) [inline insights/parsers/sysconfig.py]: Should be `NetconsoleSysconfig`.

File: insights/parsers/tests/test_sysconfig_netconsole.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from insights.parsers.netconsole import NetConsole
+from insights.parsers.sysconfig import NetconsoleSysconifg
```
Comment (bfahr) [inline insights/parsers/tests/test_sysconfig_netconsole.py]: Should be `NetconsoleSysconfig`.  Same change on line 30 as well.

File: insights/parsers/sysconfig.py [hunk 4]
```diff
@@ -351,6 +425,83 @@ class PrelinkSysconfig(SysconfigOptions):
...
+    pass
+
+
+@parser(Specs.up2date)
+class Up2DateSysconfig(SysconfigOptions):
+    """
+    Class to parse the ``/etc/sysconfig/rhn/up2date``
+
+    Typical content example::
+        serverURL[comment]=Remote server URL
```
Comment (bfahr) [inline insights/parsers/sysconfig.py]: I believe you need a blank line above this line.

File: insights/parsers/sysconfig.py (hunk 1 above)
Comment (xiangce) [reply insights/parsers/sysconfig.py]: @bfahr - Thank you for the careful review. They are fixed.

**Pull request merged**